app/main.py

```diff
@@ -256,55 +262,48 @@ def manage_login_modal(n_clicks, username, password):
         form_feedback.append(html.P("Il semble qu'il manque votre nom d'utilisateur et/ou votre mot de passe."))
 
         # The login modal remains open; other outputs are updated with arbitrary values
-        return True, form_feedback, '', [10, 10], 3
+        return True, form_feedback, [10, 10], 3
 
     else:
-        # We open the encrypted file
-        with open(os.path.join(os.path.dirname(__file__), "data/login_correspondences.enc"), 'rb') as file:
-            enc_file = file.read()
-
-        # We decrypt the file with the password stored in the git-ignored .env file
-        csv_text = decrypt(cfg.LOGIN_CORRESPONDENCES_KEY, enc_file).decode('utf8')
-
-        # We create a temporary csv out of the decrypted string
-        correspondences = StringIO(csv_text)
 
-        # We read the resulting csv as a Pandas DataFrame
-        correspondences = pd.read_csv(correspondences)
+        login_route_url = 'http://pyronear-api.herokuapp.com/login/access-token'
 
-        if username not in correspondences['username'].values or password not in correspondences['password'].values:
-            # If either the username or the password is not found in the corresponding field of the login corresponden-
-            # ces csv file, the condition is verified
+        data = {
+            'username': username,
+            'password': password
+        }
 
-            # We add the appropriate feedback
-            form_feedback.append(html.P("Nom d'utilisateur ou mot de passe erroné."))
+        response = requests.post(login_route_url, data=data).json()
 
-            # The login modal remains open; other outputs are updated with arbitrary values
-            return True, form_feedback, '', [10, 10], 3
+        check = ('access_token' in response.keys())
 
-        elif password != correspondences[correspondences['username'] == username]['password'][0]:
-            # If the password provided does not correspond to the username in the csv file, the condition is verified
+        if not check:
+            # This if statement is verified if credentials are invalid
 
             # We add the appropriate feedback
             form_feedback.append(html.P("Nom d'utilisateur ou mot de passe erroné."))
 
             # The login modal remains open; other outputs are updated with arbitrary values
-            return True, form_feedback, '', [10, 10], 3
+            return True, form_feedback, [10, 10], 3
 
         else:
             # All checks are successful and we add the appropriate feedback
             # (although the login modal does not remain open long enough for it to be readable by the user)
             form_feedback.append(html.P("Vous êtes connecté, bienvenue sur la plateforme Pyronear !"))
 
+            # For now the group_id is not fetched, we equalize it artificially to 1
+            group_id = 1
```

Comment:
For now, in the same HTTP request: no
But I think on the API side, we're gonna add a "get my access" route : passing the token, it sends you back all non sensitive info (login, scope, group_id). Either way, consider that it will be done within the next days!